app/controllers/questions_controller.rb

```diff
@@ -1,11 +1,32 @@
 class QuestionsController < ApplicationController
 
     def index
+      if flash[:params].blank?
         @questions = Question.all
+        @question = Question.new
+      else
+        @questions = Question.all
+        @question = Question.new(flash[:params])
+      end
```

Comment:
`render` を使用する前提で次の形式としましょう。
```suggestion
      @questions = Question.order(id: :desc)
      @question = Question.new
```